<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代码生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <widget class="QWidget" name="page_13"/>
        <widget class="QWidget" name="page_14"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>650</height>
         </size>
        </property>
        <widget class="QWidget" name="page_1">
         <widget class="QScrollArea" name="scrollArea_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>353</x>
              <y>50</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是数据库配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QScrollArea" name="scrollArea_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>733</width>
              <height>602</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;数据库配置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>请检查要生成接口项目代码的目标数据库，生成器将针对此数据库生成一个分层设计的接口项目</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <widget class="QWidget" name="tab">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <attribute name="title">
                 <string>数据表</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QTableWidget" name="tableWidget_DB">
                   <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>表名</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>加密字段</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>逻辑删除字段</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>业务主键</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>业务主键生成规则</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <property name="minimumSize">
                 <size>
                  <width>725</width>
                  <height>453</height>
                 </size>
                </property>
                <attribute name="title">
                 <string>视图</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QTableWidget" name="tableWidget_View">
                   <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>新建列</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>接口查询字段</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QScrollArea" name="scrollArea_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>380</x>
              <y>160</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是代码生成页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>350</x>
              <y>70</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是视图表配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>70</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是数据库表配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_last">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>上一步</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>下一步</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1507</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
